jerry.sh

```diff
@@ -627,11 +628,19 @@ update_status() {
 }
 
 update_score() {
-    status_choice=$(printf "CURRENT\nCOMPLETED\nPAUSED\nDROPPED\nPLANNING" | launcher "Filter by status: ")
-    case "$1" in
-        "ANIME") get_anime_from_list "$status_choice" ;;
-        "MANGA") get_manga_from_list "$status_choice" ;;
-    esac
+    if [ "$2" = "immediate" ]; then
+        [ "$1" = "ANIME" ] && total="$episodes_total"
+        [ "$1" = "MANGA" ] && total="$chapters_total"
+        if [ $((progress + 1)) != $(echo "$total" | cut -d " " -f1) ]; then
```

Comment:
why the cut?

Reply:
because it has the episode number and the rating

Reply:
ah ok. that was probably a bug, which i should've fixed in your fork now